package lab1_exceptions;

public class ExceptionsLab1 {

	public static void main(String[] args) {
		
		testMethodA(null);
		testMethodB();

	}


public static void testMethodA(String a){
	try{
		System.out.println("Try block OK: " + a);
	}
	catch (Exception e){
		System.out.println("Catch Exception" + e);
	}
	finally{
		System.out.println("OK from finally");
	}
}

@SuppressWarnings("finally")
public static String testMethodB(){
	try{
		return "OK";
	}
	catch (Exception e){
		return "Exception: " + e;
	}
	finally{
		return "Finally section";
	}
}
}
